### ISSUE TITLE
_output_ should strip trailing forward slashes

### ISSUE THREAD

**Issue by codingo** [codingo/Interlace#56]
When building command files I notice that the trend is to build them as:

```
nmap -sC -sV -oA _output_/scans/nmap _target_
```
Noting that a slash after `_output_` is the trend.

The problem with this is that it's not uncommon to then launch this command file as something like:

```
interlace -cL example.il -t 127.0.0.1 -o ./
```

This results in a path that has `//` within it. To better support such use cases, as a default the output flag should strip any trailing forward slashes. This should also be noted within README.md as expected behavior.

**Comment by prodigysml** [comment]
Good thinking there! Didn't realise that. Definitely worth doing.

**Comment by codingo** [comment]
Closed with #57
